ios/testmanagerd/xcuitestrunner.go

```diff
@@ -236,7 +236,7 @@ func RunXCUITest(bundleID string, testRunnerBundleID string, xctestConfigName st
 	}
 	info, err := getAppInfos(bundleID, testRunnerBundleID, apps)
```

Comment:
lets rather have `getAppInfos` do it a bit different. At the moment we are getting the infos for both the app under test and the test runner at the same time. Lets split this and get only one of them at a time. The we can simply not call it when not needed, and we are still able to handle errors if there's something wrong with the provided bundle id

Reply:
Good idea. Pushed an update, please review.